{"name": "创建英雄失败的测试用例-姓名不符合要求", "status": "passed", "parameters": [{"name": "fail_for_name", "value": "101.1"}, {"name": "success_for_volume", "value": "99"}, {"name": "success_for_power", "value": "500"}], "start": 1684150562196, "stop": 1684150562196, "uuid": "992d7e46-ffe5-41c1-91f5-a87a51d691e6", "historyId": "8e31e2cdc90f746c442452aa1a7d6dc0", "testCaseId": "df48d656dc2b7ede1a0a4026708ca17a", "fullName": "test_create_hero_homework_01.TestHeroCreate#test_create_hero_fail_for_name", "labels": [{"name": "tag", "value": "run(order=2)"}, {"name": "suite", "value": "test_create_hero_homework_01"}, {"name": "subSuite", "value": "TestHeroCreate"}, {"name": "host", "value": "LAPTOP-0EHG7NSH"}, {"name": "thread", "value": "48660-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_create_hero_homework_01"}]}